src/Classes/Particles.ts

```diff
@@ -29,7 +30,7 @@ export class Particles {
         const container = this.container;
         const options = container.options;
 
-        if (options.polygon.type === PolygonMaskType.inline) {
+        if (options.polygon.type === PolygonMaskType.inline && options.polygon.inlineArrangement === PolygonMaskInlineArrangement.onePerPoint) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5146458)